cat/CaT-service.yaml

```diff
@@ -1780,18 +1780,18 @@ paths:
             - buyer
       summary: Add a supplier to an event
       
-  /tenders/projects/{proc-id}/events/{event-id}/suppliers/{supplier-id}:
-    parameters:
-      - $ref: '#/components/parameters/ProcurementParam'
-      - $ref: '#/components/parameters/EventParam'
-      - $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Params.yaml#/components/parameters/SupplierParam'
-
-
     delete:
       description: Remove a supplier from an event
       tags:
         - Define Procurement Event
       operationId: removeSuppliers
+      requestBody:
```

Comment:
This gives an error in Swagger editor - `Delete operations cannot have a requestBody`

Reply:
Just trying to decide how much this is needed. Alternative is to make PUT replace instead of add. Worries that someone could PUT one supplier and delete all the others when they intended to add one to the existing list

Reply:
Decided to remove. Delete will need multi calls because delete doesn't use request body.  If this is a problem we could use query params if required.